components/types/modals.ts

```diff
@@ -1 +1 @@
-export type ModalSelector = 'confirm' | 'congrats';
+export type ModalSelector = 'confirm' | 'congrats' | '';
```

Comment:
why `| ''`?

Reply:
Previously a certain line would be be ` const [openModal, setOpenModal] = useState<ModalSelector | false>(false);`. With an empty string option within `ModalSelector` it's simpler.

But I've just noticed that I've only edited it within `FriendtechProvider.tsx` and it's in the old way in ETH, ERC20 and ERC721 Providers. Should I revert that change, or introduce it in the remaining providers for consistency?

Reply:
i like an empty string over `false` as the default state so we don't mix types